coreUtilities/build.gradle

```diff
@@ -41,6 +41,7 @@ dependencies {
 
     // JCommander
     compileOnly group: 'org.jcommander', name: 'jcommander'
+    implementation group: 'org.yaml', name: 'snakeyaml'
```

Comment:
why do you need snakeyaml over jackson?  They both have a healthy stream of CVE's associated with them, but AFAIK currently, we don't have this as a dependency and we do have a dependency in pretty much every project on jackson.  Can we keep out maintenance costs down & stick to only jackson for yaml needs too?

Reply:
This is a prototype of what it would take to move in this direction where I quickly jumped to snakeyaml, make sense to use jackson when we implement the change.